readthedocs/projects/views/private.py

```diff
@@ -92,7 +93,11 @@ def validate_primary_email(self, user):
             notification.send()
 
     def get_queryset(self):
-        return Project.objects.dashboard(self.request.user)
+        builds = Build.objects.filter(
+            project=OuterRef('pk'), type='html', state='finished')
+        sub_query = Subquery(builds.values('success')[:1])
+        return Project.objects.dashboard(self.request.user).annotate(
+            Count('builds'), build_success=sub_query)
```

Comment:
I think the name of the parameter should be `latest_build_success` rathar than `build_success` as its giving the latest build status.

Reply:
what about having the `Count('builds')` also in parameter like `builds_count=Count('builds')`

Reply:
sure. I'll update the PR accordingly